pkgs/development/python-modules/captum/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pythonOlder,
+  buildPythonPackage,
+  pytestCheckHook,
+  setuptools,
+  matplotlib,
+  numpy,
+  packaging,
+  torch,
+  tqdm,
+  flask,
+  flask-compress,
+}:
+
+buildPythonPackage rec {
+  pname = "captum";
+  version = "0.7.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
```

Comment:
```suggestion
```

This is not really useful, especially if it is a version way older than the ones we officially support.